katago_server/trainings/models/network.py

```diff
@@ -75,7 +78,7 @@ class Meta:
         storage=network_data_storage,
         max_length=200,
         null=False,
-        blank=False,
+        blank=True,
```

Comment:
? why change this ?

Reply:
A yes for the first net.
Then also change the `null = False` constraint.

Reply:
I read online that null = False does not work properly for FileField, and therefore setting is not a good idea? https://docs.djangoproject.com/en/dev/ref/models/fields/#null

Reply:
(setting blank=True only does work for me, I was able to create an appropriate random net)

Reply:
Interesting: I always did the distinction. "" means a string that can be empty but null is stronger and assert the willingness to put no data, not just an empty string.

I must do some research.

Reply:
Fine for me